packages/logging/system-test/logging.js

```diff
@@ -369,6 +369,22 @@ describe('Logging', function() {
       }, 1000);
     });
 
+    it('should preserve order for sequential write calls', function(done) {
+      var messages =  ['1', '2', '3', '4', '5'];
+
+      messages.forEach(function(message) {
+        log.write(log.entry(message));
+      });
+
+      setTimeout(function() {
+        log.getEntries({ pageSize: messages.length }, function(err, entries) {
+          assert.ifError(err);
+          assert.deepEqual(entries.reverse().map(prop('data')), messages);
+          done();
```

Comment:
This `done()` wasn't in place earlier, but don't worry-- after adding, the test still passed :)